### ISSUE TITLE
plugin contribution

### ISSUE THREAD

**Issue by iaon** [xebialabs-community/xld-tibcobw-plugin#1]
Hi,

I'm almost done with my version of xld-tibcobw-plugin
https://github.com/iaon/xld-tibcobw-plugin

I've did some enhancements such as deploy in fault tolerance mode or map of strings as configuration for Tibco Ears

Unfortunately my version is not compatible, I had to change some basic types for fault tolerance mode deploy.
When I will finish documentation I'll be ready to send pull request.

Do you need my contribution to your repo?

**Comment by bmoussaud** [comment]
Hi @iaon 
Thank you for your interest in this xld-tibcobw-plugin and I'm looking forward to reading your improvements. 
Can you detail the incompatibility(ies) you have introduced ?

**Comment by iaon** [comment]
Hi @bmoussaud 

I've added new type tibco.Node and it takes part of functionality  from tibco.Domain. In domain configuration we can have management server where we run Appmanage tool for configuration and some nodes where applications run.

In other words for old environment  we have to add tibco.Node object near tibco.Domain after plugin upgrade.

Also I've added "deploy generated configuration" step. It allows define TIBCO parameters directly from XLD dictionary without template XML artifact. But I's a new feature and should be compatible.
